pkg/scheduler/reconciler_test.go

```diff
@@ -61,11 +61,11 @@ func (ft *fakeTracker) Get(gvr schema.GroupVersionResource, ns, name string, opt
 	return ft.ObjectTracker.Get(gvr, ns, name, opts...)
 }
 
-func (ft *fakeTracker) Update(gvr schema.GroupVersionResource, obj runtime.Object, ns string, opts ...metav1.UpdateOptions) error {
```

Comment:
Going to file this away in my mental bank of "reasons the fake client is not what you want".